src/components/nano/ModalConfirmUnregister.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+import { t } from 'ttag';
+import $ from 'jquery';
+
+export default function ModalConfirmUnregisterNanoContract({ onClose, ncId, success }) {
+  useEffect(() => {
+    $('#modalConfirmUnregister').modal('show');
+    $('#modalConfirmUnregister').on('hidden.bs.modal', onClose);
+
+    return () => {
+      $('#modalConfirmUnregister').modal('hide');
+      $('#modalConfirmUnregister').off();
```

Comment:
suggestion(non-blocking): If we pass the modal selector to `onClose(id)` this procedure is done by the GlobalModal manager.

Reply:
Much better. Done, thanks https://github.com/HathorNetwork/hathor-wallet/pull/625/commits/1eb97caf38dffe6a3feafdf94db3825369f6e550